resources/packs/ms365/ms365.lr

```diff
@@ -146,47 +166,46 @@ msgraph.beta.rolemanagement.roledefinition {
   rolePermissions []dict
   templateId string
   version string
-  assignments() []msgraph.beta.rolemanagement.roleassignment
+  assignments() []msgraph.rolemanagement.roleassignment
 }
 
-// Microsoft Graph Beta Role Assignment
-msgraph.beta.rolemanagement.roleassignment {
+// Microsoft Graph Role Assignment
+msgraph.rolemanagement.roleassignment {
   id string
   roleDefinitionId string
   principalId string
   principal dict
 }
 
-// Microsoft Graph Beta Device Management
-msgraph.beta.devicemanagement {
-  deviceConfigurations() []msgraph.beta.devicemanagement.deviceconfiguration
-  deviceCompliancePolicies() []msgraph.beta.devicemanagement.devicecompliancepolicy
+// Microsoft Graph Device Management
+msgraph.devicemanagement {
+  deviceConfigurations() []msgraph.devicemanagement.deviceconfiguration
+  deviceCompliancePolicies() []msgraph.devicemanagement.devicecompliancepolicy
 }
 
-// Microsoft Graph Beta Device Configuration
-msgraph.beta.devicemanagement.deviceconfiguration {
+// Microsoft Graph Device Configuration
+msgraph.devicemanagement.deviceconfiguration {
   id string
   lastModifiedDateTime time
-  roleScopeTagIds []string
-  supportsScopeTags bool
   createdDateTime time
   description string
   displayName string
   version int
+  // deprecated, kept for backwards compatability.
   properties dict
 }
 
-// Microsoft Graph Beta Device Compliance Policy
-msgraph.beta.devicemanagement.devicecompliancepolicy {
+// Microsoft Graph Device Compliance Policy
+msgraph.devicemanagement.devicecompliancepolicy {
   id string
   createdDateTime time
   description string
   displayName string
   lastModifiedDateTime time
-  roleScopeTagIds []string
   version int
-  properties dict
   assignments []dict
+  // deprecated, kept for backwards compatability.
+  properties dict
```

Comment:
we should make all properties a lazy function. This allows us to properly return an error.